src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  max-width: calc(7 * (#{$cell-size} + #{$border-width} + #{$gap} + #{$padding}));
+
+  &--start-day-mon {
+    .calendar__cell:nth-child(1) {
+      margin-right: calc((6 * #{$cell-size}) + (6 * #{$border-width}) + (6 * #{$gap}) + (2 * #{$padding}));
+      width: #{$cell-size};
+      height: #{$cell-size};
+    }
+  }
+
+  &--month-length-28 {
+    @include generate-calendar-cells(28);
+  }
+
+  &--month-length-29 {
+    @include generate-calendar-cells(29);
+  }
+
+  &--month-length-30 {
+    @include generate-calendar-cells(30);
+  }
+
+  &--month-length-31 {
+    @include generate-calendar-cells(31);
+  }
```

Comment:
make sense to move these blocks to loop
you can do something like this:
<img width="937" alt="Screenshot 2023-09-17 at 17 24 52" src="https://github.com/mate-academy/layout_calendar/assets/64746943/3715f4a7-f9cb-472c-b6c0-b29c2a1d4b20">